tests_zemu/tests/bls.test.ts

```diff
@@ -0,0 +1,86 @@
+/** ******************************************************************************
+ *  (c) 2020 Zondax GmbH
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************* */
+
+import Zemu from '@zondax/zemu'
+import InternetComputerApp from '@zondax/ledger-icp'
+import { DEFAULT_OPTIONS, DEVICE_MODELS_BLS } from './common'
+
+jest.setTimeout(180000)
+
+describe('Bls', function () {
+  test.concurrent.each(DEVICE_MODELS_BLS)('verify_with_custom_key', async function (m) {
+    const sim = new Zemu(m.path)
+    try {
+      await sim.start({ ...DEFAULT_OPTIONS, model: m.name, startText: m.name === 'stax' ? '' : 'Computer' })
```

Comment:
I think you can use the isTouchDevice here. I tested and it worked

Reply:
i did not work :-(